lib/cryptoexchange/exchanges/getbtc/services/market.rb

```diff
@@ -28,7 +28,6 @@ def adapt(output, market_pair)
           ticker.last      = NumericHelper.to_d(ticker_data['last_price'])
           ticker.high      = NumericHelper.to_d(ticker_data['max'])
           ticker.low       = NumericHelper.to_d(ticker_data['min'])
-          ticker.volume    = NumericHelper.divide(NumericHelper.to_d(ticker_data['total_btc_traded']), ticker.last)
```

Comment:
@Swingcloud by removing this we wont have any information on volume, is there no way to convert it?

Reply:
seems no way, I've check this website, maybe we can do sum up all trades to get the single volume of currency, what do you think?

Reply:
We can use `total_btc_traded`